src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -188,14 +206,6 @@ public override async Task<string> Execute(OrchestrationContext innerContext, st
                 innerContext.ErrorDataConverter = this.errorDataConverter;
             }
 
-            this.Config.TraceHelper.FunctionStarting(
```

Comment:
Can you tell us more about the change to remove these log statements? Is it because they're overly verbose and/or not meaningful?

Reply:
These events are very useful and the verbosity is appropriate.

I simply moved them into the EntityMiddleware method so they are emitted at a point of time that makes more sense. In particular, FunctionStarted is now emitted before we execute the operations, not after we already executed them.